### ISSUE TITLE
训练一直 ap为0，还有loss是nan的已经把lr改成很小了0.000125，单gpu训练

### ISSUE THREAD

**Issue by lling-2021** [PaddlePaddle/PaddleDetection#6618]
[08/10 10:30:54] ppdet.engine INFO: Epoch: [9] [360/437] learning_rate: 0.000125 loss_xy: nan loss_wh: nan loss_obj: nan loss_cls: nan loss: nan eta: 10:58:56 batch_cost: 0.2701 data_cost: 0.0811 ips: 29.6175 images/s
[08/10 10:30:59] ppdet.engine INFO: Epoch: [9] [380/437] learning_rate: 0.000125 loss_xy: nan loss_wh: nan loss_obj: nan loss_cls: nan loss: nan eta: 10:57:21 batch_cost: 0.2523 data_cost: 0.0597 ips: 31.7112 images/s
[08/10 10:31:04] ppdet.engine INFO: Epoch: [9] [400/437] learning_rate: 0.000125 loss_xy: nan loss_wh: nan loss_obj: nan loss_cls: nan loss: nan eta: 10:55:55 batch_cost: 0.2744 data_cost: 0.0550 ips: 29.1509 images/s
[08/10 10:31:10] ppdet.engine INFO: Epoch: [9] [420/437] learning_rate: 0.000125 loss_xy: nan loss_wh: nan loss_obj: nan loss_cls: nan loss: nan eta: 10:54:28 batch_cost: 0.2718 data_cost: 0.0697 ips: 29.4309 images/s
[08/10 10:31:17] ppdet.utils.checkpoint INFO: Save checkpoint: output/yolov3_darknet53_270e_voc
[08/10 10:31:17] ppdet.engine INFO: Eval iter: 0
[08/10 10:31:19] ppdet.engine INFO: Eval iter: 100
[08/10 10:31:22] ppdet.engine INFO: Eval iter: 200
[08/10 10:31:24] ppdet.engine INFO: Eval iter: 300
[08/10 10:31:27] ppdet.engine INFO: Eval iter: 400
[08/10 10:31:29] ppdet.engine INFO: Eval iter: 500
[08/10 10:31:32] ppdet.engine INFO: Eval iter: 600
[08/10 10:31:34] ppdet.engine INFO: Eval iter: 700
[08/10 10:31:37] ppdet.engine INFO: Eval iter: 800
[08/10 10:31:39] ppdet.engine INFO: Eval iter: 900
[08/10 10:31:42] ppdet.metrics.metrics INFO: Accumulating evaluatation results...
[08/10 10:31:42] ppdet.metrics.metrics INFO: mAP(0.50, 11point) = 0.00%
[08/10 10:31:42] ppdet.engine INFO: Total sample number: 1000, averge FPS: 40.33762743732353
[08/10 10:31:42] ppdet.engine INFO: Best test bbox ap is 0.000.

**Comment by ghostxsl** [comment]
可否提供一下你运行的环境及命令嘛？

**Comment by lling-2021** [comment]
> 可否提供一下你运行的环境及命令嘛？
首先谢谢！
我是根据https://aistudio.baidu.com/aistudio/projectdetail/2307736?channelType=0&channel=0 里面做的，数据也是上面提供的，我用了aistudio上面好几个数据集都是这个问题
命令是：nohup python  tools/train.py -c configs/yolov3/yolov3_darknet53_270e_voc.yml --eval --use_vdl=True --vdl_log_dir="./outputfall" >logfallHelmet&
环境如下：
Package            Version
------------------ ---------
astor              0.8.1
Babel              2.10.3
bce-python-sdk     0.8.74
brotlipy           0.7.0
certifi            2022.6.15
cffi               1.15.0
chardet            5.0.0
charset-normalizer 2.1.0
click              8.1.3
colorama           0.4.5
cryptography       37.0.1
cycler             0.11.0
Cython             0.29.32
decorator          5.1.1
et-xmlfile         1.1.0
filelock           3.7.1
Flask              2.2.1
Flask-Babel        2.0.0
Flask-Cors         3.0.10
fonttools          4.34.4
future             0.18.2
gast               0.3.3
idna               3.3
importlib-metadata 4.12.0
itsdangerous       2.1.2
Jinja2             3.1.2
joblib             1.1.0
kiwisolver         1.4.4
lap                0.4.0
MarkupSafe         2.1.1
matplotlib         3.5.2
mkl-fft            1.3.1
mkl-random         1.2.2
mkl-service        2.4.0
motmetrics         1.2.5
numpy              1.23.1
opencv-python      4.6.0.66
openpyxl           3.0.10
opt-einsum         3.3.0
packaging          21.3
paddledet          2.4.0
paddlepaddle-gpu   2.3.1
paddleslim         2.2.1
paddlex            2.1.0
pandas             1.4.3
Pillow             9.2.0
pip                22.1.2
protobuf           4.21.4
pycocotools        2.0.4
pycparser          2.21
pycryptodome       3.15.0
pyOpenSSL          22.0.0
pyparsing          3.0.9
PySocks            1.7.1
python-dateutil    2.8.2
pytz               2022.1
PyYAML             6.0
pyzmq              23.2.0
requests           2.28.1
scikit-learn       0.23.2
scipy              1.8.1
setuptools         61.2.0
Shapely            1.8.2
six                1.16.0
sklearn            0.0
terminaltables     3.1.10
threadpoolctl      3.1.0
tqdm               4.64.0
typeguard          2.13.3
urllib3            1.26.11
visualdl           2.3.0
Werkzeug           2.2.1
wheel              0.37.1
xmltodict          0.13.0
zipp               3.8.1

**Comment by lling-2021** [comment]
> 可否提供一下你运行的环境及命令嘛？
按照这个上面运行的https://aistudio.baidu.com/aistudio/projectdetail/3496364?channelType=0&channel=0

**Comment by kaixin-bai** [comment]
这个issue没有更新了吗？
